gnucash/report/reports/standard/advanced-portfolio.scm

```diff
@@ -79,28 +79,16 @@ by preventing negative stock balances.<br/>")
      (gnc:make-multichoice-option
       gnc:pagename-general optname-price-source
       "d" (N_ "The source of price information.") 'pricedb-nearest
-      (list (vector 'pricedb-latest
-		    (N_ "Most recent")
-		    (N_ "The most recent recorded price."))
-	    (vector 'pricedb-nearest
-		    (N_ "Nearest in time")
-		    (N_ "The price recorded nearest in time to the report date."))
-	    )))
+      (list (vector 'pricedb-latest (N_ "The most recent recorded price"))
+            (vector 'pricedb-nearest (N_ "The price recorded nearest in time to the report date")))))
```

Comment:
Suggested names:
"Most recent"
"Closest to  report date" (or "Nearest to report date")